MTM-Frontend/src/components/Form/SpecificItemDialog/SpecificItemsDialog.tsx

```diff
@@ -84,6 +87,7 @@ export const SpecificItemsDialog = ({
 
   return (
     <div id="SpecificItems-Dialog">
+      <ErrorMessage error={error} setError={setError} />
```

Comment:
You probably want to do something like `error & <ErrorMessage error={error} setError={setError} />` so that the message won't show up when there's no error